conmon/conmon.c

```diff
@@ -80,6 +80,18 @@
 		syslog(LOG_INFO, "conmon <ninfo>: " fmt " \n", ##__VA_ARGS__); \
 	} while (0)
 
+#define ndebug(s)                                             \
+	do {                                                 \
```

Comment:
Might be better to include the `if(opt_debug)` in the macros so all debug prints will only happen if debug is set.